Pear/Controller/LoginViewController.swift

```diff
@@ -132,10 +134,21 @@ extension LoginViewController: GIDSignInDelegate {
               // Only allow users with Cornell emails or internal users (Pear email) to log in
               email.contains("@cornell.edu") || email == "cornellpearapp@gmail.com" else {
             GIDSignIn.sharedInstance().signOut()
+            do {
+                try Auth.auth().signOut()
```

Comment:
It's been a while, but can you please remind me why we use both `GIDSignIn` and `Auth`?

Reply:
GIDSignIn doesn't automatically add the user to the authenticated table in Firebase. By having auth, we can check for a user's authenticated status in Firebase database when writing database rules.